### PR TITLE
[meshmodel] Add Environment Component Def

### PR DESCRIPTION
**Pull request by theBeginner86**
**Notes for Reviewers**

Currently its under Meshery model.

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/meshmodel/meshery/1.0.0/Environment.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "logoURL": "",
+    "model": "meshery",
+    "modelDisplayName": "Meshery",
+    "primaryColor": "#FFDE00",
+    "published": true,
+    "secondaryColor": "#FFDE00",
+    "shape": "tag",
+    "styleOverrides": "{\"height\":\"22px\",\"width\":\"22px\",\"x\":\"3.5px\",\"y\":\"7.5px\"}",
+    "subCategory": "App Definition and Development",
+    "svgColor": "<?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE svg><svg xmlns=\"http://www.w3.org/2000/svg\" stroke=\"currentColor\" fill=\"currentColor\" stroke-width=\"0\" viewBox=\"0 0 24 24\" height=\"20\" width=\"20\"><path xmlns=\"http://www.w3.org/2000/svg\" fill=\"none\" d=\"M0 0h24v24H0z\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z\"></path></svg>",
```
Comment (theBeginner86) [inline server/meshmodel/meshery/1.0.0/Environment.json:19]: we need to update these svg

File: server/meshmodel/meshery/1.0.0/Environment.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
.../2000/svg\" d=\"M31.51,10.34A67.89,67.89,0,0,0,10.1,31.89L31.51,44.25Z\" class=\"cls-2\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M99.43,8A67.23,67.23,0,0,0,69.49,0V25.15Z\" class=\"cls-1\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M0,69.87A67.27,67.27,0,0,0,8.07,99.63L29.76,87.07Z\" class=\"cls-1\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M8.07,35.37A67.16,67.16,0,0,0,0,65L29.79,47.91Z\" class=\"cls-1\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M35.78,127.13A67.13,67.13,0,0,0,65.47,135V110.15Z\" class=\"cls-2\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M124.92,32a67.9,67.9,0,0,0-21.28-21.52V44.3Z\" class=\"cls-1\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M103.64,124.54A68,68,0,0,0,125,102.86L103.64,90.52Z\" class=\"cls-1\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M135,64.81a67.06,67.06,0,0,0-8-29.35L105.49,47.88Z\" class=\"cls-2\"></path><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M69.49,135a67.12,67.12,0,0,0,29.63-7.83L69.49,110Z\" class=\"cls-1\"></path><polygon xmlns=\"http://www.w3.org/2000/svg\" points=\"31.51 83.44 31.51 51.56 3.83 67.43 31.51 83.44\" class=\"cls-2\"></polygon></svg>",
```
Comment (theBeginner86) [inline server/meshmodel/meshery/1.0.0/Environment.json:32]: need to update these colors

File: server/meshmodel/meshery/1.0.0/Environment.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "displayName": "Environment",
+  "format": "JSON",
+  "metadata": {
+    "genealogy": "",
+    "isAnnotation": true,
+    "isNamespaced": false,
+    "logoURL": "",
+    "model": "meshery",
+    "modelDisplayName": "Meshery",
+    "primaryColor": "#FFDE00",
```
Comment (theBeginner86) [inline server/meshmodel/meshery/1.0.0/Environment.json:13]: this color

**Comment by theBeginner86** [comment]
Added entry in Meshery Integrations [Spreadsheet](https://docs.google.com/spreadsheets/d/1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw/edit#gid=0).

**Review by aabidsofi19 — APPROVED** [review]

**Pull request merged**